base/reflection.jl

```diff
@@ -199,14 +199,14 @@ function _dump_function(f, t::ANY, native, wrapper, strip_ir_metadata)
         str = ccall(:jl_dump_function_asm, Any, (Ptr{Void},), llvmf)::ByteString
     else
         str = ccall(:jl_dump_function_ir, Any,
-                    (Ptr{Void}, Bool), llvmf, strip_ir_metadata)::ByteString
+                    (Ptr{Void}, Bool, Bool), llvmf, strip_ir_metadata, dump_module)::ByteString
     end
 
     return str
 end
 
-code_llvm(io::IO, f::Function, types::ANY, strip_ir_metadata=true) =
-    print(io, _dump_function(f, types, false, false, strip_ir_metadata))
+code_llvm(io::IO, f::Function, types::ANY, strip_ir_metadata=true, dump_module=false) =
```

Comment:
The options should probably be documented too - and if there's going to be more than one of them, should they be kwargs?